packages/web-runtime/src/App.vue

```diff
@@ -34,7 +34,11 @@
       @confirm-secondary="modal.onConfirmSecondary"
       @mounted="focusModal"
       @beforeDestroy="focusModal"
-    />
+    >
+      <template v-if="modal.customContent" #content>
+        <p v-html="modal.customContent"></p>
```

Comment:
`<div>` might be more fitting here, as custom content could be anything: `<div>`, `<a>`, `<img>` etc. - all tags, that semantically shouldn't be placed inside a `<p>` tag.

```suggestion
        <div v-html="modal.customContent" />
```